content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,55 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers:
+
+**From a user to a controller**: 
+
+First the user signals to the controller that it should take ownership of a
+field, usually by setting some other field that the controller observes. The
+controller then claims ownership of the field by applying a configuration
+containing the field. This can be done in one of the following ways:
+
+- If the controller wants to immediately set the value to the field, it can
+apply with force set to true.
+- If the controller does not want to change the value to the field immediately,
+the controller should genereate a unique, temporary, field manager name, and
```

Comment:
It's the user that needs to do this step, not the controller. We can't assume controllers will be up to date.

Reply:
True... fixing.

Reply:
fixed